packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/CheckboxStyle.java

```diff
@@ -62,9 +71,15 @@ public Builder color(String color) {
             return this;
         }
 
+        public Builder labelPlacement(String labelPlacement) {
+            this.labelPlacement = Objects.requireNonNull(labelPlacement);
+            return this;
+        }
+
         public CheckboxStyle build() {
             CheckboxStyle checkboxStyle = new CheckboxStyle();
             checkboxStyle.color = this.color; // Optional on purpose
+            checkboxStyle.labelPlacement = Objects.requireNonNull(this.labelPlacement);
```

Comment:
See comment above about the mandatory aspect of the attribute and change this accordingly if we choose to set the attribute optional.